### ISSUE TITLE
[A1] Linha 1524 com os nomes das suas variantes errados

### ISSUE THREAD

**Issue by stgarmm** [carrismetropolitana/gtfs#444]
**Descreva o erro**  
Olá, reparei que a linha `1524 | Tercena (Estação) via Casal do Cotão - Taguspark` tem o nome das suas variantes iguais, devendo de se proceder à seguinte correção:
- A variante `1524_0_1` deverá de se chamar Tercena (Estação) via Casal do Cotão, São Marcos, Taguspark e Talaíde (Escola Secundária);
- A variante `1524_0_2` deverá de se chamar Tercena (Estação) via Casal do Cotão, São Marcos e Taguspark

**Área(s) afetada(s)**  
Quais as áreas afetadas pelo erro identificado:

- [x] Área 1 (Viação Alvorada)
- [ ] Área 2 (Rodoviária de Lisboa)
- [ ] Área 3 (TST)
- [ ] Área 4 (Alsa Todi)

**Valor(es) esperado(s)**  
Se possível, uma descrição clara e concisa da situação sem erro / situação corrigida.

**Screenshots**  
Se fizer sentido, adicione imagens que ajudem a explicar o problema.

**Contexto adicional**  
Adicione qualquer outra informação útil à resolução do problema.

**Comment by DiogoBaptista98** [comment]
É uma linha circular, logo é normal que tenha nome igual como quase todas o tem. A nível pratico também não concordo com as propostas pela enorme extensão do nome que sugere.

**Comment by stgarmm** [comment]
Sim, agora reprando bem, o nome fica extenso, propronho então que se altere a `1514_0_1` para Tercena (Estação) via Casal do Cotão - Taguspark e Talaíde (Escola Secundária) e se deixe a variante `1514_0_2` com o nome em uso (`Tercena (Estação) via Casal do Cotão - Taguspark`)
